samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js

```diff
@@ -230,20 +226,21 @@ const StorybookDecorator = (story, config) => {
 		skinKnobs.skin = select('skin', skins, Config, currentSkin);
 	}
 	const {accent, highlight} = !allSkins && boolean('default skin styles', Config) ? defaultColors[skinKnobs.skin] : {};
-	if (config.parameters && config.parameters.props) {
-		config.props = config.parameters.props;
-	}
+
+	// NOTE: 'config' object is not extensible
+	const hasInfoText = config.parameters && config.parameters.info && config.parameters.info.text;
+	const hasProps = config.parameters && config.parameters.props;
 
 	return (
 		<Theme
-			title={`${config.kind} ${config.story}`.trim()}
-			description={config.description}
+			title={`${config.kind}`.replaceAll('/', ' ').trim()}
```

Comment:
```suggestion
			title={`${config.kind}`.replace(/\//g, ' ').trim()}
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll
replaceAll() is supported since Chrome 85. I suggest that use replace() with regex.